genisys/modules/nat.py

```diff
@@ -67,4 +67,9 @@ def install_location(self: Self) -> Path:
 
     # end install_location
 
+    def setup_commands(self: Self) -> Union[List[str], List[List[str]]]:
+        return ["sudo netfilter-persistent save", "sudo netfilter-persistent reload", "sudo systemctl enable iptables", "sudo systemctl start iptables"]
```

Comment:
Have you done testing for what each of these commands does? I was under the impression `netfilter-persistent save` saved the currently active config, which I'd worry would overwrite the config file you generated. Maybe not a concern RN but something to look into during E2E testing.

Reply:
Changed first command to the correct command to load config from file into currently running config.